examples/kopia/main.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2024 The Kanister Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+
+	"github.com/kanisterio/safecli/examples/kopia/args"
+	"github.com/kanisterio/safecli/examples/kopia/repository"
+)
+
+func main() {
+	args := repository.CreateArgs{
```

Comment:
Can we have some doc for this example? Like what we're trying to achieve and how do we do that?

Reply:
Good point. I will add a second kopia command and some docs.